facturx/flavors/factur-x/xml/basic.xml

```diff
@@ -1,165 +1,165 @@
 <rsm:CrossIndustryInvoice xmlns:qdt="urn:un:unece:uncefact:data:standard:QualifiedDataType:100" xmlns:ram="urn:un:unece:uncefact:data:standard:ReusableAggregateBusinessInformationEntity:100" xmlns:rsm="urn:un:unece:uncefact:data:standard:CrossIndustryInvoice:100" xmlns:udt="urn:un:unece:uncefact:data:standard:UnqualifiedDataType:100" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
     <rsm:ExchangedDocumentContext>
         <ram:BusinessProcessSpecifiedDocumentContextParameter>
-            <ram:ID>A1</ram:ID>
+            <ram:ID/>
         </ram:BusinessProcessSpecifiedDocumentContextParameter>
         <ram:GuidelineSpecifiedDocumentContextParameter>
             <ram:ID>urn:factur-x.eu:1p0:basic</ram:ID>
         </ram:GuidelineSpecifiedDocumentContextParameter>
     </rsm:ExchangedDocumentContext>
     <rsm:ExchangedDocument>
-        <ram:ID>2017-TEST-04</ram:ID>
-        <ram:TypeCode>380</ram:TypeCode>
+        <ram:ID/>
+        <ram:TypeCode>0</ram:TypeCode>
         <ram:IssueDateTime>
-            <udt:DateTimeString format="102">20171031</udt:DateTimeString>
+            <udt:DateTimeString format="102">00000000</udt:DateTimeString>
```

Comment:
Why so many 00 and not an empty tag?

Reply:
Because date has 8 digits?

Reply:
Why not use an empty tag, like you did with all other tags? You said that's good practice. Why add placeholders for this one?